### PR TITLE
remove redundant parts of worker dockerfile

### PR DESCRIPTION
**Pull request by rcooke-warwick**
This will reduce the size of the container, and will also remove an error we were getting when trying to create a new release

Change-type: patch
Signed-off-by: Ryan Cooke <ryan@balena.io>

### CODE REVIEW

File: worker/Dockerfile.template [hunk 1]
```diff
@@ -55,10 +33,6 @@ RUN sed -i 's/^#cgroup_controllers =.*/cgroup_controllers = []/g' /etc/libvirt/q
 
 COPY --from=go-build /go/bin/glider /usr/local/bin
 
-COPY --from=firmware /src/usbsdmux/dist/usbsdmux-configure /usr/local/bin
-COPY --from=firmware /src/teensy_loader_cli/teensy_loader_cli /usr/local/bin
-COPY firmware/StandardFirmataPlus.ino.hex .
-COPY firmware/SDcardSwitch.ino.hex .
```
Comment (klutchell) [inline worker/Dockerfile.template:61]: What were these binaries used for previously?

Comment (rcooke-warwick) [reply worker/Dockerfile.template:61]: I think they were used for the very first iteration of the testbot/autohat board many years ago.

**Review by klutchell — APPROVED** [review]
lgtm

**Review by vipulgupta2048 — APPROVED** [review]
Tested ✅

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Pull request merged**